pkg/manifests/manifests.go

```diff
@@ -2098,6 +2103,19 @@ func (f *Factory) MetricsServerDeployment(kubeletCABundle *v1.ConfigMap, tlsSecr
 		return dep, nil
 	}
 
+	if err := validateAuditProfile(config.Audit.Profile); err != nil {
+		return nil, err
+	}
+
+	profile := strings.ToLower(string(config.Audit.Profile))
+	containers[idx].Args = append(containers[idx].Args,
+		fmt.Sprintf("--audit-policy-file=/etc/audit/%s-profile.yaml", profile),
+		"--audit-log-path=/var/log/metrics-server/audit.log",
```

Comment:
I'd send them directly to stdout for more visibility, also to avoid reserving 500MB of disk (for a Deployment it could be considered a lot).
And if they'll not be enabled all the time (my other proposition),  they'll not harm the logging stack.

Reply:
Since we will be including it in must-gather we will be keeping it in storage